### PR TITLE
fix observables 2.2

### PR DESCRIPTION
**Pull request by zoemccormick**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

This PR fixes the observables setup and README for release-2.2. It updates the kibana-observables-proxy configuration to use (new) pathogen templates on branch release-2.2.

It also adds two new flags - one at `global.data.external.uses3`, and one `global.rbac.edge` that takes a boolean value on whether or not to configure rbac in the default way on the edge.

<br/><br/>

### CODE REVIEW

File: observables/README.md [hunk 1]
```diff
@@ -8,108 +8,170 @@ Simply put:
...
+mkdir observables/certs
+openssl pkcs12 -in certs/quickstart.p12 -cacerts -nokeys -out observables/certs/ca.crt
+openssl pkcs12 -in certs/quickstart.p12 -clcerts -nokeys -out observables/certs/user.crt
+openssl pkcs12 -in certs/quickstart.p12 -nocerts -nodes -out observables/certs/user.key
+```
+
+Then, to run the dashboarder service:
+
+```bash
+docker run --rm -v $(pwd)/observables/certs:/usr/local/dashboarder -e GREYMATTER_URL=https://$GREYMATTER_API_HOST/services/kibana-observables-proxy/7.1.0/api/saved_objects/ docker.greymatter.io/internal/dashboarder generate greymatter
```
Comment (chrisbsmith) [inline observables/README.md:149]: 🥳  Glad to see that this docker image is working!!

File: observables/README.md [hunk 2]
```diff
@@ -135,7 +197,7 @@ Once this is done, make the necessary [mesh updates](#mesh-updates-control-prome
 
 ## Removing Observables
 
-The make file has the ability to remove the observables deployment as a whole or individual pieces.  
+The make file has the ability to remove the observables deployment as a whole or individual pieces.
```
Comment (chrisbsmith) [inline observables/README.md:200]:
```suggestion
The makefile has the ability to remove the observables deployment as a whole or individual pieces.
```

**Review by chrisbsmith — COMMENTED** [review]
These look good @zoemccormick. I have a few comments inline below.  However, it also looks like this PR is bringing a few other changes with it (Data s3 and Edge RBAC). Is that intended?

**Comment by zoemccormick** [comment]
@chrisbsmith ah yes, sorry this is a poorly named PR. I wanted to add those two flags if approved - the default behavior is the same for both, this just adds the ability to switch on uses3 for data without having to go into the values files, and allows rbac to be turned off for edge for the case when not everyone will be using quickstart

**Review by chrisbsmith — APPROVED** [review]
LGTM, nice work!

**Pull request merged**